composer.json

```diff
@@ -46,6 +46,8 @@
         },
         "preferred-install": "dist"
     },
+    "minimum-stability": "dev",
```

Comment:
please revert this

Reply:
This line is needed to be able to install `chrome-php/wrench` 1.7. I removed it in the last commit, and now tests won't pass. I guess this will be fixed after the `wrench` release.